src/utils/errors.js

```diff
@@ -44,21 +45,20 @@ getBrowserInfo().then(
   () => {},
 );
 
-let terms = false;
-observe('terms', (value) => {
-  terms = value;
-});
+export async function captureException(error) {
+  const { terms } = await store.resolve(Options);
```

Comment:
When using observe pattern, it adds a function call to queue (or create a queue if used outside background), which is rerly used. 

If for any reason the previous pattern would fail in setting terms, we would not send error reports too. I think the situation is the same.